src/vulkan-renderer/world/cube.cpp

```diff
@@ -259,8 +259,9 @@ void Cube::set_type(const Type new_type) {
         break;
     case Type::OCTANT:
         const float half_size = m_size / 2;
+        std::uint8_t index = 0;
         auto create_cube = [&](const glm::vec3 &offset) {
-            return std::make_shared<Cube>(weak_from_this(), half_size, m_position + offset);
+            return std::make_shared<Cube>(weak_from_this(), index++, half_size, m_position + offset);
```

Comment:
Set type of sub cubes to `SOLID`.

Reply:
I think that belongs to [aboves](https://github.com/inexorgame/vulkan-renderer/pull/385#discussion_r659206714) discussion. (Otherwise please explain and unresolve.) :)